minio/progress.py

```diff
@@ -0,0 +1,96 @@
+import time
+import sys
+from threading import Thread
+from .compat import queue, queue_empty
+
+
+class Progress(Thread):
```

Comment:
Each class should be documented properly since this is written like a generic progress printer.